yoda_dbt2looker/generator.py

```diff
@@ -262,7 +262,14 @@ def lookml_dimension_groups_from_model(
         and map_adapter_type_to_looker(adapter_type, column.data_type)
         in looker_date_types
     ]
-    return date_times + dates
+
+    exposure_dimension_groups = []
```

Comment:
maybe the "pythonic" list generation is more natural here ( for all the lists you iterate on and append to a new one)